src/symbolize/symbolizer.rs

```diff
@@ -166,6 +193,19 @@ impl Builder {
         self
     }
 
+    /// Enable/disable source code location information (line numbers,
+    /// file names etc.).
+    pub fn enable_src_location(mut self, enable: bool) -> Builder {
```

Comment:
a bit of inconsistency. it's code_info and CodeInfo in Sym, but here it's called "src_location". Maybe let's pick one way of naming this?

Reply:
Yeah, I noticed that. Will change it.